packages/icons/src/svg/overview.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+  <g transform="translate(0 1.388)">
```

Comment:
this transform should be stripped, and the reflected transformation applyed directly on to the sub element position attributes